### ISSUE TITLE
Firefox download

### ISSUE THREAD

**Issue by Mghubti** [mozilla/bedrock#12116]
On Sept 9, 2022, I download Firefox. In the download menu it said Mozilla.org not Firefox download, do I went ahead and downloaded it. Now, I have a problem with my Mac Dock. Something else can with the download that is making my Mac Dock active when I started my computer this morning.

Describe the issue that you're seeing.
 I can see my Mac Dock moving to the right like something is been mover from the Mac Dock immediately.

### Steps to reproduce
Show up went I downloaded Firefox yesterday.

Clear steps describing how to reproduce the issue. Providing a URL, screenshot.
 At starting the computer, it would be hard to record a screenshot because of the quick action the Mac Dock dose.

### Expected result:
 I would like to see whatever Mozilla.org but into my Mac Dock remove immediately.

What should happen?  
 I had to removed Firefox browser from my computer this morning.  I want to reinstall Firefox again without having to see something going on with my Mac Dock reacting.

### Actual result

What happened.

### Environment

What browser and operating system are you using?
Right now Safari with DuckDuckGo

**Comment by alexgibson** [comment]
Thanks for opening this issue @Mghubti!

I'm not sure I quite fully understand the problem you're encountering based on your description.

Can I ask, can you please provide the website URL where you downloaded Firefox from?

Thank you

**Comment by Mghubti** [comment]
Hi Alex Gibson

The URL  was https://firefox.com <https://firefox.com/>
There were several downloads on the page's website and the one I picked had something in it that got into my iMac Dock. It is an intrusive app that keeps reacting when I turned on my computer every morning, and it also comes with pop-ups.
I am still having this problem and need to have removed.

I called Apple support yesterday and did an upgrade but that didn’t help. 

***@***.***

**Comment by alexgibson** [comment]
Hi, @Mghubti - if you downloaded Firefox from https://firefox.com then you can be assured that's the official build of Firefox, and you can also be confident that it does not ship with, or contain any malicious code that might interfere with your system in this way.

Can I ask, does the problem present itself even when Firefox is not running? If so, then this is most likely an issue with your macOS installation. If the problem only happens when Firefox is open, then can I suggest you try running Firefox in safe mode to see if the problem still persists? https://support.mozilla.org/en-US/kb/diagnose-firefox-issues-using-troubleshoot-mode

If you need further help, you might also try asking in our support forum: https://support.mozilla.org/en-US/questions/new/desktop

Thanks again

**Comment by Mghubti** [comment]
To mozilla/bedrock

Thanks for letting know that it was not a Firefox issue. I download Firefox then the next day I removed Firefox when I noticed a reaction on my iMac Dock. 
 
I  just had brought my computer from an Apple repair store and the Dock menu was working ok. So, when I download Firefox on my computer the Dock menu started reacting. I am still have this issue. I need to fix this issue before I download Firefox again.

Mghubti
